lib/internal/webidl.js

```diff
@@ -275,20 +277,34 @@ function createSequenceConverter(converter) {
       const val = converter(res.value, {
         __proto__: null,
         ...opts,
-        context: `${opts.context}, index ${array.length}`,
+        context: `${opts.context}[${array.length}]`,
       });
       ArrayPrototypePush(array, val);
     };
     return array;
   };
 }
 
+// https://webidl.spec.whatwg.org/#js-interface
+function createInterfaceConverter(name, prototype) {
+  return (V, opts = kEmptyObject) => {
+    // 1. If V implements I, then return the IDL interface type value that
```

Comment:
shouldn't `I` be `prototype`?

Reply:
Good suggestion. I will update the argument name instead of the comment. The comment is from the spec and I think it will be clearer if the argument is consistent with the spec.